start: Data Access and Data Versioning to mention Model in titles (#2096) (#2214)
iterative/dvc.org: content/docs/start/data-and-model-access.md

---
title: 'Get Started: Data and Model Access'
---

# Get Started: Data and Model Access

We've learned how to _track_ data files in DVC and how to commit their versions
to Git. Machine learning models are typically large files written and read by
programs. DVC can track and version model files similar to data files. The next
questions are: How can we _use_ these artifacts outside of the project? How do I
download a model to deploy it? How do I download a specific version of a model?

> How do I reuse datasets across different projects?

Reviewer: This change seems to break the semantics of the paragraph. TBH I don't think it's necessary, we already state "data and models".

I do like the correction to the first sentence, "... , and how to commit their versions to Git" but the next 2 sentences are out of context here.

> We've learned how to _track_ data files in DVC and how to commit their versions
> to Git. Machine learning models are typically large files written and read by
> programs. DVC can track and version model files similar to data files. The next
> questions are: How can we _use_ these artifacts outside of the project? How do I
> download a model to deploy it? How do I download a specific version of a model?
> How do I reuse datasets across different projects?

Reviewer: ```suggestion
Okay, we've learned how to _track_ data and models with DVC, and how to commit
their versions to Git. The next questions are: How can we _use_ these artifacts
outside of the project? How do I download a model to deploy it? How to download
a specific version of a model? Or reuse datasets across different projects?
```

---
title: 'Get Started: Data and Model Access'
---

# Get Started: Data and Model Access

Okay, now that we've learned how to _track_ data files in DVC and how to version

> them with Git. _Models_ in a machine learning project are also files written and

Reviewer: > Okay, now that we've ... with Git.

Incomplete sentence

Author: Actually _version_ is a verb there. Fixed in 09dc8ca7c498ee0414c3b6bce7430cc82054f012. Thanks.

Reviewer: Still incomplete. "to version" was OK. The part that is messing the sentence up is "now that". Please either remove that or complete the sentence after "in Git, (something)" (or rewrite in some other complete way). Thanks

Author: Removed `Okay, now that` in b46cca31

> read by programs and DVC can track and version them similar to data files.

Reviewer: Maybe "Machine learning models typically are also large files in the project. DVC can ..."

Author: Should this also mention "Data Science"? I'm also thinking about SEO.

Author: Revised in 7731587eb58c58805f5ec7c5f398dc385b1dce9c.

Reviewer: > Should this also mention "Data Science"? I'm also thinking about SEO.

Was it in the desired keywords? Someone actually did research to come up with those. I see they're gone now (prob. should've left that comment until right before merging this)

Reviewer: I see this still reads "Models in a machine learning project". Let's use "Machine learning models" please (SEO).

Author: Keywords are in BC documents, this is a GS document. There were no keywords here 😄

Reviewer: > Keywords are in BC documents

True haha, nvmd.

> Next question is how can we _use_ these artifacts outside of the project? How do

Reviewer: Would be ideal to keep a single paragraph here.

Reviewer: Also, "The next questions are ..."

Author: Fixed in a3b15ba4e4a463fe47884150fc25238b85853cbe. Thanks. 👍🏼

Reviewer: Thanks. But it's still missing "The".

Author: Fixed in b46cca31
